pkg/artifactory/resource_artifactory_user.go

```diff
@@ -68,14 +68,12 @@ func resourceArtifactoryUser() *schema.Resource {
 			"disable_ui_access": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
```

Comment:
The Artifactory data schema has a default value, let's set them here (as well as for `internal_password_disabled` attribute.